mne/io/bti/read.py

```diff
@@ -26,17 +25,6 @@ def _unpack_simple(fid, dtype, out_dtype):
     return out
 
 
-def read_str(fid, count=1):
-    """ Read string """
-    dtype = np.dtype('>S%i' % count)
-    string = fid.read(dtype.itemsize)
-    data = np.fromstring(string, dtype=dtype)[0]
-    bytestr = b('').join([data[0:data.index(b('\x00')) if
-                          b('\x00') in data else count]])
-
-    return str(bytestr.decode('ascii'))  # Return native str type for Py2/3
-
-
```

Comment:
I moved this function to utils. This should have probably been private in the first place so deprecation is not needed?
ping @dengemann